### PR TITLE
build: create Travis CI config

### PR DESCRIPTION
**Pull request by AndersDJohnson**
Adding a config for Travis CI - a popular free-for-open-source continuous integration system to run tests on every PR. Refer to https://docs.travis-ci.com/user/languages/javascript-with-nodejs/ for more details.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+language: node_js
+node_js:
+  - node
+  - lts/*
```
Comment (AdarshKonchady) [inline .travis.yml:4]: Can we add the test command here?

Comment (AndersDJohnson) [reply .travis.yml:4]: @akonchady We could, but it should default to `npm test` per https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#default-build-script which we've defined here:
https://github.com/akonchady/disable-eslint-issues-cli/blob/ba487e36d5c12420753e806bb64e76e3f3088164/package.json#L10

Comment (AdarshKonchady) [reply .travis.yml:4]: Good to know 👍

**Review by AdarshKonchady — APPROVED** [review]
👍

**Comment by AndersDJohnson** [comment]
@akonchady Thanks! You can go ahead and configure the repo on Travis CI under your account if you want. I've configured it on my fork to test it and it's working great - check it out https://travis-ci.org/AndersDJohnson/disable-eslint-issues-cli

**Comment by AdarshKonchady** [comment]
Thanks @AndersDJohnson . Added travis CI badge here - https://github.com/akonchady/disable-eslint-issues-cli/pull/12

**Pull request merged**